docs/usage/config-validation.md

```diff
@@ -48,3 +48,22 @@ $ renovate-config-validator first_config.jsonn
 
 You can create a [pre-commit](https://pre-commit.com) hook to validate your configuration automatically.
 Go to the [`renovatebot/pre-commit-hooks` repository](https://github.com/renovatebot/pre-commit-hooks) for more information.
+
+### Let Renovate check your config file as you're writing it
+
+After using Renovate for a while you likely want to change Renovate's config.
+Renovate can check your new config as you're writing it, but you must put your work in a branch with a _special_ name.
+
+Follow these steps:
+
+1. Create a new Git branch that matches the `{{branchPrefix}}reconfigure` pattern.
+   For example, if you're using the default prefix `renovate/`, your branch name must be `renovate/reconfigure`. Commit your updated Renovate config file to this branch, and push it to your Git hosting platform.
+
+2. Each time Renovate runs, it searches for a branch that matches the reconfigure pattern and validates the Renovate configuration file.
+
+3. Renovate will check the config file on the reconfigure branch, even if it has a different name, compared to the config file on the default branch.
+   So feel free to change the Renovate config file name.
+
+4. Depending on the outcome of the Renovate config validation run, Renovate will add a passing or failing check to the branch.
+
+5. If there's an open pull request from the _reconfigure_ branch then Renovate will comment in that PR and list all the validation errors. Commit and push your fixes to the branch, and try again. When the config validation passes, you can merge your Renovate config.
```

Comment:
```suggestion
5. If there's an open pull request from the _reconfigure_ branch then Renovate will comment in that PR with details in case there are validation errors. Each commit will be revalidated the next time Renovate runs on the repository.
```